main.js

```diff
@@ -22,6 +24,22 @@ const plugins = [
 async function main() {
     try {
         const currentBranch = process.env.GITHUB_HEAD_REF;
+
+        const extraPlugins = core.getMultilineInput('semanticReleasePlugins').filter(Boolean);
+        if (extraPlugins.length > 0) {
+            console.info('Installing extra plugins...');
+            for (const extraPlugin of extraPlugins) {
+                console.info(`Installing ${extraPlugin}...`);
+                const pluginName = extractPluginName(extraPlugin);
+                await executeCommand(`yarn add ${extraPlugin}`, __dirname);
```

Comment:
does this error if `yarn` is not installed?

Reply:
Yes, but we already require `yarn` in [`index.js`](https://github.com/fingerprintjs/action-semantic-release-info/blob/main/index.js#L25) file.

Reply:
got it, nvm